cvmfs/nfs_shared_maps.cc

```diff
@@ -206,20 +208,17 @@ bool GetPath(const uint64_t inode, PathString *path) {
   path->Assign(raw_path, strlen(raw_path));
   sqlite3_reset(stmt_get_path_);
   pthread_mutex_unlock(&lock_);
-  atomic_inc64(&dbstat_inode_found_);
+  perf::Inc(n_dbstat_inode_found_);
   return true;
 }
 
 
 string GetStatistics() {
```

Comment:
It is used in the line 259 of the nfs_maps.cc file:

```
string GetStatistics() {
  if (use_shared_db_)
    return nfs_shared_maps::GetStatistics();

  string result = "Total number of issued inodes: " +
                  StringifyInt(seq_-root_inode_) + "\n";

  string stats;
  db_inode2path_->GetProperty(leveldb::Slice("leveldb.stats"), &stats);
  result += "inode --> path database:\n" + stats + "\n";

  db_path2inode_->GetProperty(leveldb::Slice("leveldb.stats"), &stats);
  result += "path --> inode database:\n" + stats + "\n";

  return result;
}
```

Shall we remove that if (used_shared_db_) ?